config.go

```diff
@@ -33,6 +34,7 @@ type PluginConfig struct {
 	Timeout    duration
 	Interval   duration
 	Dimensions []*Dimension
+	Service    string
```

Comment:
PluginConfig had better to have a `Destination` field. It's value may be `CloudWatch` or `Mackerel`.